setup.cfg

```
@@ -15,5 +15,10 @@ python_requires = >=3.6
 
 [options]
 zip_safe = False
-packages = find:
 install_requires = numpy
+packages=find:
```

This is deprecated/renamed and will raise a `Package would be ignored"` warning during installation.

```suggestion
packages = find_namespace:
```

---

This is not how I understand https://setuptools.pypa.io/en/latest/userguide/package_discovery.html#custom-discovery. `find:` is for standard packages and `find_namespace:` for namespace packages

---

If pip doesn't complain, it doesn't look like in my local test, I will also not.